addons/escoria-core/game/core-scripts/esc_game.gd

```diff
@@ -27,6 +35,11 @@ export(EDITOR_GAME_DEBUG_DISPLAY) var editor_debug_mode = \
 		EDITOR_GAME_DEBUG_DISPLAY.NONE setget _set_editor_debug_mode
 
 
+# Ready function
+func _ready():
+	escoria._on_settings_loaded(escoria.settings)
```

Comment:
This shouldn't call a private method. Please change it to public or workaround it by using signals.

Reply:
Renamed this function to `apply_settings()`